content/telegraf/v1.16/about_the_project/release-notes-changelog.md

```diff
@@ -8,14 +8,31 @@ menu:
     parent: About the project
 ---
 
+## v1.16.1 [2020-10-28]
+
+### Input plugin updates
+
+- Apache Kafka Consumer (`kafka_consumer`): Add Kafka SASL-mechanism authentication support for SCRAM-SHA-256, SCRAM-SHA-512, and GSSAPI.
+- Microsoft SQL Server (`sqlserver`):
+  - Fix a syntax error of the Azure queries.
+  - Remove synthetic performance counters that no longer exist from the `sqlserver_performance_counters` measurement.
+  - Add a new tag (`sql_version_desc`) to identify the readable SQL Server version.  
+- RAS (`ras`):
+  - Disable on specific Linux architectures (MIPS64, mips64le, ppc64le, riscv64).
+  - Fix an issue to properly close file handlers.
+- Processes (`processes`): Fix an issue with receiving `no such file or directory` stat error.
+- Windows Performance Counters (`win_perf_counters`): Fix an issue with the counter with a negative denominator error to cause gathering operations to fail.
```

Comment:
is there a way to rephrase without double with?

Reply:
> 
> 
> is there a way to rephrase without double with?

Fix an issue with the counter where a negative denominator error would cause gathering operations to fail.

Reply:
Thank you, @PrplHaz4!